### ISSUE TITLE
Accessibility linting

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/ProjectVisBug#198]
would be nice if the a11y inspector went further than showing failing color contrast, but could recognize nodes by type and specify missing props or strategies that can help that be better. essentially going the next step by setting expectations and general requirements for certain scenarios.

**Comment by AnnieTaylorCHEN** [comment]
I tried this one on Linuxmint with Chromium. Am I supposed to use Alt + Click then the tag will be pinned (fixed) to where I clicked? Or did I understand it wrong? Or did I use the wrong key? Currently nothing is pinned anywhere when I move my mouse the tag I intended to pin will disappear from view. Thanks!

**Comment by argyleink** [comment]
@AnnieTaylorCHEN sounds like a linux bug with pinning accessibility cards? i'll investigate tomorrow and let you know if i can repro and/or how to solve it 👍

**Comment by argyleink** [comment]
you found a bug, thank you! logged it here #251
